shopfloor/actions/data.py

```diff
@@ -125,6 +125,10 @@ def _package_parser(self):
             "name",
             "shopfloor_weight:weight",
             ("package_storage_type_id:storage_type", ["id", "name"]),
+            (
+                "quant_ids:total_quantity",
+                lambda rec, fname: sum(rec.quant_ids.mapped("quantity")),
+            ),
```

Comment:
Due to the mass test, performance problems are gradually appearing. One performance problem is the data loading. Because of that, we should only load the data that is really needed, in the screen. With this change, total_quantity is always loaded for the package data, also in other scenarios where it is not needed.

Reply:
We could create a separate method, like _get_package_parser which is getting as argument the record to load and kwargs. And there we can add a kwarg like with_total_quantity.

Reply:
can we do this improvement in another PR?

Reply:
yes we can do this. 
I think we should create a PR which is doing it for all scenarios then.